distribution/lib/Standard/Table/0.0.0-dev/src/Internal/Fan_Out.enso

```diff
@@ -0,0 +1,281 @@
+from Standard.Base import all
+
+import project.Data.Column.Column
+import project.Data.Table.Table
+import project.Data.Type.Value_Type.Value_Type
+import project.Internal.Problem_Builder.Problem_Builder
+
+from project.Errors import Column_Count_Exceeded, Column_Count_Mismatch
+from project.Internal.Java_Exports import make_string_builder
+
+polyglot java import org.enso.table.data.mask.OrderMask
+
+## PRIVATE
+   Transform a table by transforming a column into a set of columns. Takes a
+   function that maps a single element of the input column to a vector of output
+   values. The original column is replaced by the new columns.
+
+   Arguments:
+   - table: The table to transform.
+   - input_column: The column to transform.
+   - function: A function that transforms a single element of `input_column`
+     to multiple values.
+fan_out_to_columns : Table -> Text | Integer -> (Any -> Vector Any) -> Integer | Nothing -> (Integer -> Any) -> Problem_Behavior -> Table | Nothing
```

Comment:
This was taken from Split_Tokenize, and has been in use in that so far. The only change was to optionally generalize it to use other column builders.

Reply:
Thanks for this comment - it makes it easier to review!

I wonder if there are any diff tools out there that could detect chunks of code being moved between files - it would be so much easier in such cases :)

Reply:
I didn't think so, but I just looked and apparently git does it!

```
git diff --color-moved develop
```

If you run that on this PR, the code moved to Fan_Out.enso shows the modified lines in a different color than the just-copied ones.

Reply:
Woow so cool! Thanks!